app/views/users/index.html.erb

```diff
@@ -1 +1,5 @@
-<h1>Placeholder for /users and root</h1>
\ No newline at end of file
+<div class="users">
+<% @users.each do |user| %>
+  <%= render partial: 'user', locals: { user: user } %>
+<% end %>
+</div>
```

Comment:
- [ ] Kindly add some margin and padding and border to your css so as to match your design with the given design

Given design
![image](https://github.com/LeslieAine/blog-app1/assets/81932478/30ee2a25-292f-45b9-8a82-96853eeee442)

Your design
![image](https://github.com/LeslieAine/blog-app1/assets/81932478/70361356-9b35-4bca-92d5-3393f87b383b)